bin/run-tests

```diff
@@ -91,21 +91,21 @@ main() {
         check_exit "$?"
     fi
 
-    if [[ "$opt" == "frontend" || -z "$opt" ]]; then
-        # Test frontend
-        command="yarn --cwd frontend run test:ci --colors"
-        if [ $verbose == "false" ]; then
-          command="yarn --cwd frontend run test:ci --silent --colors"
-        fi
-        echo
-        log "Running frontend tests"
-        if [[ $docker == "true" ]]; then
-          docker exec test-frontend bash -c "$command"
-        else
-          $command
-        fi
-        check_exit "$?"
-    fi
+    # if [[ "$opt" == "frontend" || -z "$opt" ]]; then
+    #     # Test frontend
+    #     command="yarn --cwd frontend run test:ci --colors"
+    #     if [ $verbose == "false" ]; then
+    #       command="yarn --cwd frontend run test:ci --silent --colors"
+    #     fi
+    #     echo
+    #     log "Running frontend tests"
+    #     if [[ $docker == "true" ]]; then
+    #       docker exec test-frontend bash -c "$command"
+    #     else
+    #       $command
+    #     fi
+    #     check_exit "$?"
+    # fi
```

Comment:
Do we still need these blocks? (same for line 83)

Reply:
whoops